src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Paste.java

```diff
@@ -52,46 +53,64 @@ public void run() {
                         sender.sendMessage("Warning: ConsoleSpamFix installed! Please disable it before reporting any errors!");
                         return;
                     } else {
-                        if (!cmdArg[0].equals("--force")) {
+                        if (Arrays.stream(cmdArg).noneMatch(str -> str.contains("--ignore-csf"))) {
```

Comment:
we should also provide the command usage as well

Reply:
Actually, it never works in my test, for an unknown reason.